src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+instance, of a _contradiction_.” (Hegel 1991, 716). The “concept” of Mechanism
+is the logical moment with which the section, A. The Mechanical Object finished.
+The `mechanical object` is now a negative unity: the negativity of having the
+identical determinateness to other mechanical objects whilst being indifferent
+to them. This is the contradiction with which A. The Mechanical Object
+concludes. This contradiction is now explored further in the (a) The Formal
+Mechanical Process.
+
+Above, Hegel writes that it is the “positing” of the contradiction in the
+`mechanical object` that gets the mechanical process started and it is worth
+dwelling on this for a moment. Hegel explains that by “positing” he means,
+
+> It follows from the Notion just indicated that the interaction of objects
+> takes the form of the _positing_ of the _identical_ relation of the objects.
+> This consists merely in giving to the determinateness that is determined, the
+> form of _universality_; this is _communication_, which does not involve
+> transition into an opposite (Hegel 1991, 716) Thus, the “positing” is an
+> expression of the logical interaction of objects that relate to each other as
+> identical. But where has this logical interaction come from? If we cast our
+> minds back to the conclusion of A. The Mechanical Object we will recall that
+> the contradiction was more specifically conceptualised as a negative unity.
+> This negativity is the interaction of the `mechanical object` with another
+> `mechanical object` – whereby a `mechanical object` has both an identical
+> determinateness to another `mechanical object`, whilst being indifferent to
+> it.
+
+The negativity of the contradiction, then, logically unfolds as the positing of
+their identical determinateness. At this point, one might legitimately ask: why
+is the first positing that of their identity and not that of their indifference
+to each other? The answer must be: because it is their identity that binds them
+into a relation with one another. It is because they have an identical
+determinateness that there is a negative _unity_ in the first place, and it is
+because of their identical determinateness that an interaction can take place.
+Thus, the identical relation is posited.
+
+The logical result of this positing is that the identical determinateness of the
+`mechanical objects` is itself determined. In other words, the logical structure
+of that identity is posited and in being posited is hardened (determined) into
+an actual moment between `mechanical objects`. Their identity is no longer just
+one side of a contradiction, but it has its day in the Sun, it is determined as
+a moment of what it is for the `mechanical object` to be in a formal mechanical
+relation with another `mechanical object`.
+
+Accordingly, because their identity is determined, their relation takes the form
+of a `universal` – there is nothing other than `mechanical objects` and these
```

Comment:
Why is there nothing other than mechanical objects? Provide some context to the reader here.

Reply:
The footnote would take the reader to the previous section which would elaborate on this. With that said, do you still think I should expand on this here?

Reply:
Yeah, that should be fine. If a footnote is only a link, make the link inline with the text. Otherwise, as a user, I have to click first on the footnote, then on the link.

Reply:
I say something similar below, I think it works better as a footnote. Because, having the footnote in the text breaks the flow. It isn't a necessary part of the text and only somebody who wants further clarification will scroll down to click on the link.